### PR TITLE
MC-13776 Add docs for rollbacks

### PR DESCRIPTION
**Pull request by razine-bensari**
#### Changes made

- Added docs for list, get, create for rollbacks

### CODE REVIEW

File: source/includes/administration/_rollbacks.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          "id": "4fa42796-6b93-4d9f-b5f7-34c4d1c796ea",
+          "type": "swiftstack"
+        }
+      }
+    ],
+    "organization": {
+      "name": "Operator",
+      "id": "1b47df69-6458-4ea6-8c4b-2894b3f7ec00"
+    },
+    "name": "Sasuke",
```
Comment (razine-bensari) [inline source/includes/administration/_rollbacks.md]:
```suggestion
    "name": "Rollback",
```

File: source/includes/administration/_rollbacks.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    "description": "Konoha",
```
Comment (razine-bensari) [inline source/includes/administration/_rollbacks.md]:
```suggestion
    "description": "Fix bug in pipeline for date conversion",
```

File: source/includes/administration/_rollbacks.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+            "id": "2d468fc0-434d-49a7-9739-e8d4fda0c488",
+            "type": "stackpath"
+          }
+        }
+      ],
+      "organization": {
+        "name": "Operator",
+        "id": "1b47df69-6458-4ea6-8c4b-2894b3f7ec00"
+      },
+      "name": "HElloWorld",
```
Comment (razine-bensari) [inline source/includes/administration/_rollbacks.md]:
```suggestion
      "name": "Rollback January",
```

File: source/includes/administration/_rollbacks.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
+## Rollback Triggers
```
Comment (jasondias9) [inline source/includes/administration/_rollbacks.md]:
```suggestion
## Rollbacks
```

File: source/includes/administration/_rollbacks.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+A rollback trigger enabled a reseller to process past usage data with a new pricing configuration. Pricing configuration can change due to a pricing change, a modified product or a change in pricing packages.
```
Comment (jasondias9) [inline source/includes/administration/_rollbacks.md]:
```suggestion
Rollbacks enable service providers to reprocess usage data from the past. 
Currently the only supported rollback type REPROCESS uses existing collected usage metrics and reprocesses it through the monetization engine. The most current effective pricing configuration will be used to price these records. 
```

File: source/includes/administration/_rollbacks.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+You need to specify a reset date, the affected service connection(s) and the targeted organization(s). Pending invoices will be regenerated.
+A rollback will be created for each service connection and organization pairs that are applicable.
```
Comment (jasondias9) [inline source/includes/administration/_rollbacks.md]: This seems out of place given we are not talking about the create yet.

File: source/includes/administration/_rollbacks.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+| `resetDate`<br/>_string_     | The date in which we want to start the rollback (inclusive).                                                                         |
+| `rollbacks`<br/>_Array_      | A list of rollbacks scoped to a service connection and organization that were generated based on the rollback trigger configuration. |
+
+<!-------------------- CREATE A ROLLBACK TRIGGER -------------------->
+
+### Create a rollback trigger
+
+`POST /rollbacks`
+
+Creates a rollback trigger for the reseller org using the provided set of service connection IDs and organization IDs.
```
Comment (jasondias9) [inline source/includes/administration/_rollbacks.md]:
```suggestion
Creates a rollback which will result in reprocessing of usage for the given request body. 
This API currently supports the REPROCESS rollback type which uses existing usage previously collected by plugin usage collectors and re processes it in the monetization engine. The latest pricing configuration will be used when processing the records. 

The API body supports targeting a set of service connections and organizations as well as the date from which the rollback should begin. 
```

**Review by jasondias9 — CHANGES_REQUESTED** [review]
looking good overall! I left some suggestions

File: source/includes/administration/_rollbacks.md (hunk 6 above)
Comment (razine-bensari) [reply source/includes/administration/_rollbacks.md]: [`c4b2589` (#446)](https://github.com/cloudops/cloudmc-api-docs/pull/446/commits/c4b2589605c613c4e62f4219722851be04f7fbe3)

**Review by jasondias9 — APPROVED** [review]

**Pull request merged**